src/main/java/com/paneedah/mwc/MWC.java

```diff
@@ -65,6 +65,15 @@ public final class MWC {
     @SidedProxy(serverSide = "com.paneedah.mwc.proxies.CommonProxy", clientSide = "com.paneedah.mwc.proxies.ClientProxy")
     public static CommonProxy commonProxy;
 
+    @Mod.EventHandler
+    public void construction(FMLConstructionEvent constructionEvent) {
+        //Do this to avoid Groovyscript MWC Mod container getting ignored for being unused class
+        if(Loader.isModLoaded("groovyscript")){
+            new MWCContainer();
+        }
```

Comment:
Single lines if should not have curly braces
```suggestion
        if (Loader.isModLoaded("groovyscript"))
            new MWCContainer();
```